README.md

```diff
@@ -204,15 +204,15 @@ Looking for resources for code languages? Take a look at [the awesome collection
+* [poio-api ![GitHub stars](https://img.shields.io/github/stars/cidles/poio-api.svg)](https://github.com/cidles/poio-api)  Poio API is a free and open source Python library to access and search data from language documentation in your linguistic analysis workflow. It converts file formats like Elan’s EAF, Toolbox files, Typecraft XML and others into annotation graphs as defined in ISO 24612. Those graphs, for which we use an implementation called “Graph Annotation F… http://media.cidles.eu/poio/poio-api/
 * [poio-corpus ![GitHub stars](https://img.shields.io/github/stars/cidles/poio-corpus.svg)](https://github.com/cidles/poio-corpus)  The Poio Corpus is a freely available collection of language resources for the lesser-used languages. The data is extracted from free sources like Wikipedia, dictionaries, documents, websites and others.
 * [poio-doc ![GitHub stars](https://img.shields.io/github/stars/cidles/poio-doc.svg)](https://github.com/cidles/poio-doc)  Documentation of the Poio project.http://www.poio.eu
 * [poio-site ![GitHub stars](https://img.shields.io/github/stars/cidles/poio-site.svg)](https://github.com/cidles/poio-site)  The website of the Poio project - http://www.poio.eu
 * [pressagio ![GitHub stars](https://img.shields.io/github/stars/cidles/pressagio.svg)](https://github.com/cidles/pressagio)  Pressagio is a library that predicts text based on n-gram models. For example, you can send a string and the library will return the most likely word completions for the last token in the string.
 * [PrimerPro](https://github.com/sillsdev/PrimerPro) The purpose of PrimerPro is to assist the literacy worker in the development of primers for a given language. 
 * [pyannotation ![GitHub stars](https://img.shields.io/github/stars/cidles/pyannotation.svg)](https://github.com/cidles/pyannotation)  PyAnnotation is a Python Library to access and manipulate linguistically annotated corpus files.
-* [PyAnnotationGraph ![GitHub stars](https://img.shields.io/github/stars/PhonologicalCorpusTools/PyAnnotationGraph.svg)](https://github.com/PhonologicalCorpusTools/PyAnnotationGraph)  Implement the formal model for linguistic annotations described in Bird and Liberman (2001) using Python and SQL
-* [pyDelphin ![GitHub stars](https://img.shields.io/github/stars/goodmami/pydelphin.svg)](https://github.com/goodmami/pydelphin)  Python libraries for DELPH-IN (Friendly Fork)
+* [PyAnnotationGraph ![GitHub stars](https://img.shields.io/github/stars/PhonologicalCorpusTools/PyAnnotationGraph.svg)](https://github.com/PhonologicalCorpusTools/PolyglotDB)  Implement the formal model for linguistic annotations described in Bird and Liberman (2001) using Python and SQL
```

Comment:
This one isn't accurate. I don't think the original PyAnnotationGraph is here anymore - but PolyglotDB isn't the same repo, and should be added independently, while PyAnnotationGraph should be removed.